### ISSUE TITLE
SRGAN-Loss-Function

### ISSUE THREAD

**Issue by Akettle44** [Akettle44/SRGAN-PyTorch#2]
Goal: Create loss function for SRGAN
How: Create custom loss function by combining existing PyTorch Loss Functions

Design Inspiration: https://github.com/Akettle44/SenNetworks/blob/main/transformers.ipynb
Explore best way to verify loss functions in PyTorch.

Acceptance Criteria: Implement Loss Function in custom class. Create unit test to verify that loss is computed correctly.
